raiko-guest/config/raiko-guest.manifest.template

```diff
@@ -1,7 +1,16 @@
-# Copy to target/debug and run:
-#   gramine-manifest -Dlog_level=error -Darch_libdir=/lib/x86_64-linux-gnu/ raiko-guest.manifest.template raiko-guest.manifest
-#   gramine-sgx-sign --manifest raiko-guest.manifest --output raiko-guest.manifest.sgx
-#   gramine-sgx ./raiko-guest
+# This is Gramine's manifest file.
+# To learn more see: https://gramine.readthedocs.io/en/stable/manifest-syntax.html
+#
+# How to use it?
+#
+# 1. Build the app with `cargo build`.
+# 2. Copy this file `target/debug`.
+# 3. Run:
+#     export SECRETS_DIR="./secrets"
+#     export INPUT_FILES_DIR="/tmp"
```

Comment:
Notice that Gramine requires to explicitly declare which directories can be read by `raiko-guest`. Currently the input files for `raiko-guest` are assumed to be saved in `/tmp`.

Reply:
Can the host read these variables instead of cmd arguments? "cache" and "guest"

Reply:
These environment variables are being used only in this file (`raiko-guest/config/raiko-guest.manifest.template`). It is _not_ being used in any Rust code at all. That template file is then compiled into Gramine configuration file with the `gramine-manifest` command as explained [here](https://github.com/johntaiko/zeth/blob/7e5bae51cb309d8e69003f0ccf73b582f67e985a/README_Raiko.md#sgx-enabled).

Instead of using env variables we can provide these params as a `gramine-manifest`'s params if you prefer not to use env variables.